webapp/src/main/resources/application.sql.yml

```diff
@@ -3,12 +3,6 @@ sql:
     findWarehouseImports: >-
       SELECT i.id, i.digest FROM import i limit 10;
 
-  district:
-    findAllIds: >-
```

Comment:
where did the findAllIds go?  not used or moved to the common application.sql.yml?

Reply:
moved into common